Source/Common/FEXServerClient.cpp

```diff
@@ -196,7 +198,7 @@ namespace FEXServerClient {
   }
 
   int ConnectToAndStartServer(char *InterpreterPath) {
-    int ServerFD = ConnectToServer();
+    int ServerFD = ConnectToServer(false);
```

Comment:
hmm, maybe have a wrapper function like:

```cpp
// Insert appropriate name here
int ConnectToServerButBeCoolAboutIt() {
    return ConnectToServer(false);
}
```

or something, or maybe like:

```cpp
enum class ConnectOption {
    Default,
    NoPrintConnectionError,
};

int ConnectToServer(ConnectionOption option) {
  // ...
}
```

since `ConnectToServer(false)` and ConnectToServer(true)` seem a little opaque in terms of what the bool is doing

Reply:
I like the enum approach. Switched over to that.